changelogs/fix-7893-country-input-label-distorted

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: Tweak
+
+OBW: Override Country/Region label overflow style to avoid truncated descenders. #8186
```

Comment:
This changelog entry is no longer accurate.

Reply:
Yeah, I'll change it in this pass.